Example/Podfile

```diff
@@ -1,9 +1,16 @@
 source 'https://cdn.cocoapods.org/'
-platform :ios, '11.0'
+platform :ios, '16.0'
```

Comment:
Dit zorgt er voor dat het example project altijd iOS16 en hoger ondersteund. Als we iOS12 en hoger ondersteunen zou dit toch op iOS12 moeten staan zodat we het ook zo kunnen testen?